docs/UserGuide.md

```diff
@@ -544,6 +545,12 @@ Listed here are some frequently asked questions. Should you require any other as
 **Q**: Why are there no guests in my GuestBook?<br>
 **A**: If you didn't enter the `clear` command, it is likely that your `guestbook.json` file is **corrupted**, causing GuestBook to reset it to prevent any malicious files from being used.
 
+**Q**: I cannot find a guest staying on a particular date even though it should be within the dates of stay. Why is that so?
+**A**: It is likely that you entered a date within the start and end dates of the guest's stay (e.g, for a guest staying from 13-01-2022 to 15-01-2022, you entered 14-01-2022 to try and find the guest). Currently, this is **not supported** in GuestBook.
+
+**Q**: Why am I not able to add guests to the same room even though they have different dates of stay?
+**A**: As we expect guests to check in at the hotel itself, we did not support this function so that GuestBook remains simple and easy to use. As such, all the rooms must be unique to each guest in GuestBook.
```

Comment:
Maybe we can say "For this current version of GuestBook, guests are only entered when they check in at the hotel. As such, there should not be a case where the same room is occupied by different people on different dates at that point in time. As such, this feature is a safety check by GuestBook to ensure no guests are allocated the same room during their stay. In summary, to ensure GuestBook remains simple and easy to use, all rooms must be unique to each guest in GuestBook."

Reply:
Great explanation, agreed!